js/reducers/feedback-reducer.ts

```diff
@@ -61,19 +75,30 @@ export default function feedback(state = new FeedbackState({}), action: any) {
     case RECEIVE_ACTIVITY_FEEDBACKS:
       const settings: any = state.get("settings")?.toJS();
       hasScoredQuestions = state.get("hasScoredQuestions")?.toJS();
-      const actFeedbacks = action.response.reduce((map: any, feedback: any) => {
-        if (feedback.activityId) {
-          feedback.hasBeenReviewed = computeHasBeenReviewed({feedback, settings, hasScoredQuestions});
-        }
-        map[`${feedback.activityId}-${feedback.platformStudentId}`] = feedback;
-        return map;
-      }, {});
+      actFeedbacks = updateHashBeenReviewed({
+        activityFeedbackValues: action.response,
+        settings,
+        hasScoredQuestions
+      });
       return state.set("activityFeedbacks", fromJS(actFeedbacks) as Map<any, any>);
     default:
       return state;
   }
 }
 
+export const updateHashBeenReviewed = (params: {activityFeedbackValues: any; settings: any; hasScoredQuestions: any}) => {
```

Comment:
Wondering if you meant to call this updateHasBeenReviewed instead of updateHashBeenReviewed?

Reply:
Good catch - I'll update it.  How about `canIHashRevieweeeed`?